copycat/api/src/main/java/org/apache/kafka/copycat/data/CopycatSchema.java

```diff
@@ -161,11 +175,19 @@ public static void validateValue(Schema schema, Object value) {
         // Special case for bytes. byte[] causes problems because it doesn't handle equals()/hashCode() like we want
         // objects to, so we support both byte[] and ByteBuffer. Using plain byte[] can cause those methods to fail, so
         // ByteBuffers are recommended
```

Comment:
Ah, really a leftover comment that should have been removed, but I've adjusted and moved it up by the list of BYTE classes in SCHEMA_TYPE_CLASSES.